apps/matrix-conduit/nginx/matrix.conf.template

```diff
@@ -0,0 +1,19 @@
+server {
+    server_name ${APP_DOMAIN};
```

Comment:
Thanks for your Pull Request, however, this is not possible as of today to use the variables in another file than `docker-compose.yml`. I'll work on this feature once I'm done with the translations

Reply:
Thank you for review!
In that case using APP_DOMAIN var in nginx configuration is not responsiblility of runtipi.
This works according to the official Nginx container [documentation](https://github.com/docker-library/docs/tree/master/nginx#using-environment-variables-in-nginx-configuration-new-in-119).

You may check if it works correctly

Reply:
Got it, it make a lot more sense now